packages/override-tools/src/test/Manifest.test.ts

```diff
@@ -155,10 +155,208 @@ test('BaseFileNotFound', async () => {
 
   const expectedError: ValidationError = {
     type: 'baseFileNotFound',
-    file: 'foo/bar.js',
+    file: 'foo\\bar.js',
   };
 
   const testManifest = new Manifest(ourManifestData, ovrRepo, reactRepo);
   const errors = await testManifest.validate();
   expect(errors).toEqual([expectedError]);
 });
+
+test('HasOverride', async () => {
+  const manifest = new Manifest(testManifestData, ovrRepo, reactRepo);
+  expect(manifest.hasOverride(overrideFiles[0].filename)).toBe(true);
```

Comment:
>expect [](start = 2, length = 6)

nit: OOC what test framework is this? I don't see any import statement for this.

Reply:
This is Jest. It doesn't use imports, as it does some magic transforms on the file itself.